packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue

```diff
@@ -474,12 +479,19 @@ const extractConfigurationsFromInputs = async () => {
 		});
 	}
 
-	if (datasetIds.value) {
+	const newDatasetIds = difference(datasetIds.value, configExtractionDocumentIds);
```

Comment:
nitpick: add comment explaining that we only want to create new model configs for datasets that we have not already utilized to create model configs for this model